src/openrct2/actions/SetCheatAction.hpp

```diff
@@ -772,4 +780,17 @@ DEFINE_GAME_ACTION(SetCheatAction, GAME_COMMAND_CHEAT, GameActionResult)
         auto parkSetParameter = ParkSetParameterAction(isOpen ? ParkParameter::Open : ParkParameter::Close);
         GameActions::ExecuteNested(&parkSetParameter);
     }
+
+    void CreateDucks(int count) const
```

Comment:
Did you think about returning a bool, just in case it fails to create any ducks in any of the `count*100` attempts?

Reply:
We don't need error handling for cheats like that.